src/style.css

```diff
@@ -1 +1,44 @@
 /* add styles here */
+* {
```

Comment:
Using the universal selector '*' for styling is not recommended as it impacts performance. It's better to apply styles to specific elements or classes that require them.